src/aznfswatchdog

```diff
@@ -65,33 +65,60 @@ chattr -f +i $RANDBYTES
 # kernels. If we find an entry stuck for more than a certain time in SYN_SENT state
 # we delete the entry so that kernel looks up fresh NAT rules and creates a new entry.
 #
-declare -A cthash_synsent
-declare -A cthash_synsent_now
+declare -A cthash_stuck
+declare -A cthash_stuck_now
 
 reconcile_conntrack_one()
 {
     local l_ip=$1
     local l_sport=$2
     local l_dport=$3
-    local l_nfsip=$4
+    local state=$5
 
-    key="${l_ip}:${l_sport}:${l_dport}:${l_nfsip}"
-    let cthash_synsent[$key]++
-    let cthash_synsent_now[$key]++
+    if [ $state == "ESTABLISHED_UNREPLIED" ]; then
+        local l_sysip=$4
 
-    #
-    # We are called every 5 secs, so this deletes an entry stuck in
-    # SYN_SENT for 25/30 secs.
-    #
-    if [ ${cthash_synsent[$key]} -ge 5 ]; then
-        cmd="conntrack -D -p tcp -d $l_ip -r $l_nfsip --sport $l_sport --dport $l_dport"
-        wecho "Deleting conntrack entry stuck in SYN_SENT state [$cmd]"
+        key="${l_ip}:${l_sport}:${l_dport}:${l_sysip}"
+        let cthash_stuck[$key]++
+        let cthash_stuck_now[$key]++
 
-        eval $cmd
-        if [ $? -ne 0 ]; then
-            eecho "Failed to delete conntrack entry [$cmd]!"
-        else
-            unset cthash_synsent[$key]
+        #
+        # We are called every 5 secs, so this deletes an entry stuck in
+        # UNREPLIED for 25/30 secs.
+        #
+        if [ ${cthash_stuck[$key]} -ge 5 ]; then
+            cmd="conntrack -D -p tcp -d $l_ip -r $l_sysip --sport $l_sport --dport $l_dport --state ESTABLISHED"
```

Comment:
again, remove --state ESTABLISHED, just matching based on the other fields is enough